### PR TITLE
www-apps/hugo: add 0.125.4

### PR DESCRIPTION
**Pull request by tastytea**

### CODE REVIEW

**Review by Jamim — CHANGES_REQUESTED** [review]
_Hello @tastytea,_

First of all, thank you for keeping **Hugo** up-to-date in Gentoo Linux! 🙇🏼

I noticed that **Hugo** `0.125.4` [has been released](https://github.com/gohugoio/hugo/releases/tag/v0.125.4).
Would you mind updating the PR in favor of the latest version?

_Thanks in advance!_

**Comment by tastytea** [comment]
sure, on it!

**Review by Jamim — APPROVED** [review]

File: www-apps/hugo/hugo-0.125.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	https://github.com/gohugoio/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://tastytea.de/files/gentoo/${P}-vendor.tar.xz
+"
+
+# NOTE: To create the vendor tarball, run:
+# `go mod vendor && cd .. && tar -caf ${P}-vendor.tar.xz ${P}/vendor`
+
+LICENSE="Apache-2.0 BSD BSD-2 MIT MPL-2.0 Unlicense"
+SLOT="0"
+KEYWORDS="amd64 ~arm64 ~loong ~riscv ~x86"
```
Comment (ceamac) [inline www-apps/hugo/hugo-0.125.4.ebuild:20]:
```suggestion
KEYWORDS="~amd64 ~arm64 ~loong ~riscv ~x86"
```
I'll fix this while merging

**Review by ceamac — APPROVED** [review]

File: www-apps/hugo/hugo-0.125.4.ebuild (hunk 1 above)
Comment (tastytea) [reply www-apps/hugo/hugo-0.125.4.ebuild:20]: oops, thank you! 💖

**Pull request closed without merging**